Userland/Libraries/LibWeb/Streams/ReadableStream.h

```diff
@@ -30,7 +31,7 @@ class ReadableStream final : public Bindings::PlatformObject {
         Errored,
     };
 
-    static WebIDL::ExceptionOr<JS::NonnullGCPtr<ReadableStream>> construct_impl(JS::Realm&, Optional<JS::Handle<JS::Object>> const& underlying_source);
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<ReadableStream>> construct_impl(JS::Realm&, Optional<JS::Handle<JS::Object>> const& underlying_source, QueuingStrategy const&);
```

Comment:
Looking at the IDL definition I would expect `QueuingStrategy const&` to be `Optional<QueuingStrategy> const& = {}`

Reply:
Done and done!